README.md

```diff
@@ -98,8 +98,8 @@ The repository structure is described below.
 │
 ├── 3D                                 : Directory to store documents related to proposed 3D task group
 │   ├── README.md                      : Homepage of the 3D Imagery and Data Task Group (3DTG)
-│   ├── charter_3d_task_group_of_audubon_core_2019-06-11.docx   : Word version of submitted charter
-│   ├── charter_3d_task_group_of_audubon_core_2019-06-11.pdf    : PDF version of submitted charter
+│   ├── charter_3d_task_group_of_audiovisual_core_2019-06-11.docx   : Word version of submitted charter
+│   ├── charter_3d_task_group_of_audiovisual_core_2019-06-11.pdf    : PDF version of submitted charter
```

Comment:
same here

Reply:
Fixed in 7c67ed2

Reply:
Note: I have not reverted renaming the ac charter itself: https://github.com/tdwg/ac/pull/252/files#diff-d9bcae0ddc1c38fc3ae04f2f9b42eb57ec931c86204821975c20ddd8d304cfdf